agentinstall.sh

```diff
@@ -4,6 +4,10 @@ apt update
 echo "install docker"
 apt install -y docker.io
 
+echo "install docker-compose"
```

Comment:
@ciaranevans so there is no `apt install`?

Reply:
It would appear not